charts/posthog/values.yaml

```diff
@@ -31,11 +31,7 @@ clickhouseOperator:
   serviceType: NodePort
 
 # -- Env vars to throw into every deployment (web, worker, and plugin server)
-env:
-  - name: ASYNC_EVENT_PROPERTY_USAGE
-    value: "true"
-  - name: EVENT_PROPERTY_USAGE_INTERVAL_SECONDS
-    value: "86400"
+env: []
```

Comment:
I would add a comment here to show how to add env vars if they want to. The [] is a bit confusing.

Just an example

```yaml
# env:
#   - name: FOO
#     value: bar
```

Reply:
Note that we do have an example later in the file now, but added since I though about it earlier too